# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/moz.build
# Commit: 91cb8ee8ddd8
# Full Hash: 91cb8ee8ddd844699c7190e3d73d65ccd92934c3
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r 1086a6f8a332 -r 91cb8ee8ddd8 gfx/webrender_bindings/moz.build
--- a/gfx/webrender_bindings/moz.build	Thu Nov 05 19:28:57 2020 +0000
+++ b/gfx/webrender_bindings/moz.build	Thu Nov 05 19:26:21 2020 +0000
@@ -76,9 +76,11 @@
     EXPORTS.mozilla.webrender += [
         "DCLayerTree.h",
         "RenderCompositorANGLE.h",
+        "RenderCompositorD3D11SWGL.h",
         "RenderD3D11TextureHost.h",
     ]
     UNIFIED_SOURCES += [
+        "RenderCompositorD3D11SWGL.cpp",
         "RenderD3D11TextureHost.cpp",
     ]
     SOURCES += [
@@ -118,3 +120,4 @@
 FINAL_LIBRARY = "xul"
 
 CXXFLAGS += CONFIG["TK_CFLAGS"]
+CXXFLAGS += CONFIG["MOZ_CAIRO_CFLAGS"] 